src/view.rs

```diff
@@ -859,3 +859,87 @@ mod view_response_builder_tests {
         assert_eq!(expected, got);
     }
 }
+
+pub struct AllDocumentsViewValue {
+    rev: Revision,
+}
+
+impl AllDocumentsViewValue {
+    pub fn revision(&self) -> &Revision {
+        &self.rev
+    }
+}
+
+impl serde::Serialize for AllDocumentsViewValue {
+    fn serialize<S>(&self, serializer: &mut S) -> Result<(), S::Error>
+        where S: serde::Serializer
+    {
+        Ok(try!(serializer.serialize_struct_elt("rev", &self.rev)))
+    }
+}
+
+impl serde::Deserialize for AllDocumentsViewValue {
```

Comment:
Like 70 LOC just to deserialize a single value object..maybe you know a shorter way to do this.